iothub_client/inc/internal/iothub_internal_consts.h

```diff
@@ -9,11 +9,7 @@ extern "C"
 {
 #endif
 
-    static const char* IOTHUB_API_VERSION = "2019-10-01";
-    // TODO: https://github.com/Azure/azure-iot-sdk-c/issues/1547 tracks removing this preview 
-    // variable once the underlying logic is enabled on all IoTHubs.
-    static const char* IOTHUB_API_PREVIEW_VERSION = "2020-05-31-preview";
-
+    static const char* IOTHUB_API_VERSION = "2020-09-30";
```

Comment:
Does this have full-support for distributed tracing or do we need to use a different -preview APIversion?

Reply:
That's something that needs to be verified. Notice that "IOTHUB_API_PREVIEW_VERSION" was introduced by John for pnp, so it is not related to distributed-tracing.